src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useVirtualContributorWizard.tsx

```diff
@@ -595,6 +615,20 @@ const useVirtualContributorWizard = (): useVirtualContributorWizardProvided => {
     availableExistingSpaces,
   ]);
 
+  useEffect(() => {
```

Comment:
I have moved this to execute it just after the vcCreation, I think it's better there instad of here with a useEffect but I'm happy to discuss